src/WingetCreateCLI/Properties/Resources.resx

```diff
@@ -1099,4 +1099,7 @@
   <data name="VersionManifest_Message" xml:space="preserve">
     <value>Version Manifest:</value>
   </data>
+  <data name="UseOverrides_ErrorMessage" xml:space="preserve">
+    <value>Please try using the architecture and/or scope overrides to uniquely match new URLs to existing installer nodes in the manifest.</value>

```

Comment:
'Please' should not be used in warning or error messages
```suggestion
    <value>Try using the architecture and/or scope overrides to uniquely match new URLs to existing installer nodes in the manifest.</value>

```

Reply:
Thanks, I'll keep that in mind! Resolved.